src/content/en/2022/capabilities.md

```diff
@@ -16,7 +16,372 @@ featured_stat_2: TODO
 featured_stat_label_2: TODO
 featured_stat_3: TODO
 featured_stat_label_3: TODO
-unedited: true
 ---
 
-## TODO
+## Introduction
+
+Compelling web experiences aren't limited to basic browser capabilities; they can take advantage of their underlying operating system. Web platform APIs expose these capabilities that are the foundation for [Progressive Web Apps (PWA)](./pwa_)—web applications capable of providing high-quality experiences like platform-specific apps.
+
+In addition, some functionality on the web platform gives access to lower-level features such as access to the [file system](https://developer.mozilla.org/en-US/docs/Web/API/File_System_Access_API), [geolocation](https://developer.mozilla.org/en-US/docs/Web/API/Geolocation_API), access to the [clipboard](https://developer.mozilla.org/en-US/docs/Web/API/Clipboard_API), and even the ability to detect [gamepads](https://developer.mozilla.org/en-US/docs/Web/API/Gamepad_API).
+
+## Methodology
+
+This chapter used the HTTP Archive's public dataset of millions of pages. These pages were archived as if they were visited on both desktop and mobile, as some sites will serve different content based on what device is requesting the page.
+
+The HTTP Archive's crawler parsed the source code for all of these pages to determine which APIs were used on the pages. For instance, regular expressions, such as `/navigator\.share\s*\(/g`, test pages to see if the API is found in its source code.
+
+This method does have two significant issues. First, it may underreport some APIs used as it can not detect obfuscated code that may exist due to minification. Additionally, it may overreport occurrences of APIs because it does not execute code to see if an API is used. Regardless of these limitations, we believe this methodology should provide a sufficiently good overview of what capabilities are used on the web.
```

Comment:
```suggestion
This method does have two significant issues. First, it may underreport some APIs used as it can not detect obfuscated code that may exist due to minification, for example, when `navigator` was minified to `n`. Additionally, it may overreport occurrences of APIs because it does not execute code to see if an API is actually used. Regardless of these limitations, we believe this methodology should provide a sufficiently good overview of what capabilities are used on the web.
```